docs/quick_start_example.md

```diff
@@ -2,16 +2,15 @@ Quick Start Example
 ===================
 
 Sending actions to and getting observations from the robot is very easy.  See
-the following example that simply sends a constant position command:
+the following example, using the TriFinger robot, that simply sends a constant
+position command:
```

Comment:
i think it would be important to make a comment here that this is just an example of the frontend usage, and it will just keep waiting in this case, as no backend is started.